### ISSUE TITLE
NTR: Coxsackievirus RT-PCR in blood, stool, CSF, vesicular fluid positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8113]
New term request Coxsackievirus RT-PCR in blood, stool, CSF, vesicular fluid positive (nadea3_181017170910): Reverse transcriptase polymerase chain reaction; RNA  UMLS: 2748321

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000908
- label: Positive bloodstream Coxsackievirus nucleic acid pathogen test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of Coxsackievirus in the blood circulation by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8113
- synonyms:
